dbtasks/pom.xml

```diff
@@ -29,6 +29,11 @@
       <artifactId>liquibase-core</artifactId>
       <version>4.33.0</version>
     </dependency>
+    <dependency>
```

Comment:
Do we need this  info.picocli cli ? I think, the dependency is need to run GraalVM-enabled command line apps.

Reply:
Seems like that there was classnotfound issue.